packages/model-viewer/src/features/scene-graph/image.ts

```diff
@@ -20,34 +20,34 @@ import {Image as GLTFImage} from '../../three-components/gltf-instance/gltf-2.0.
 import {Renderer} from '../../three-components/Renderer.js';
 
 import {Image as ImageInterface} from './api.js';
-import {$gltfImage, $threeTexture, TextureInfo} from './texture-info.js';
-import {$onUpdate, $sourceObject, ThreeDOMElement} from './three-dom-element.js';
+import {$correlatedObjects, $onUpdate, $sourceObject, ThreeDOMElement} from './three-dom-element.js';
 
 
 
 const loader = new ImageLoader();
 const quadMaterial = new MeshBasicMaterial();
 const quad = new PlaneGeometry(2, 2);
 
-export const $underlyingTexture = Symbol('threeTextures');
-const $textureInfo = Symbol('textureInfo');
+export const $sourceTexture = Symbol('sourceTexture');
 export const $applyTexture = Symbol('applyTexture');
 
 /**
  * Image facade implementation for Three.js textures
  */
 export class Image extends ThreeDOMElement implements ImageInterface {
-  get[$underlyingTexture]() {
-    return this[$textureInfo][$threeTexture];
+  get[$sourceTexture]() {
+    console.assert(
```

Comment:
correlatedObjects are allowed to be null, this ensures that operations can't be applied to potential temporary objects without any backing data. I don't think this can happen in the current state of things, but overtime this might save some headaches.